proprietary/tokens/src/components/ams/logo.tokens.json

```diff
@@ -1,8 +1,9 @@
 {
   "ams": {
     "logo": {
-      "block-size": { "value": "2.5rem" },
+      "block-size": { "value": "{ams.space.grid.md}" },
```

Comment:
Why use (grid) spacing for block size?

Reply:
Not sure, I copied this from [1664](https://github.com/Amsterdam/design-system/pull/1664). @VincentSmedinga?

Reply:
It may be a bit of a stretch, but I wanted to express that the logo is exactly as tall as the vertical whitespace above and below it. Another advantage is that it assumes the same clamping.

Reply:
It depends on what we do with the Header block spacing for me. I think having the Logo be the same height as the space before and after it looks off. If we change that, maybe we should reconsider this value as well.

Reply:
We’ve changed this to use regular spacing.